src/Quiz/Quiz.js

```diff
@@ -179,4 +179,4 @@ const mapDispatchToProps = (dispatch) => ({
   setCurrentList: data => dispatch(setCurrentList(data))
 })
 
-export default connect(mapDispatchToProps, {setCurrentList})(Quiz)
+export default connect(null, mapDispatchToProps)(Quiz)
```

Comment:
I noticed you didn't have to import actions and in the dispatch call `action.setCurrentList`. Do we want to do that? It seemed to be working correctly. Was Will saying we could do that if we wanted to be more explicit with the naming so we didn't get confused or was there an actual reason?